management/src/main/java/org/kaazing/gateway/management/context/DefaultManagementContext.java

```diff
@@ -59,6 +64,10 @@
 import org.kaazing.gateway.management.gateway.GatewayManagementBeanImpl;
 import org.kaazing.gateway.management.gateway.GatewayManagementListener;
 import org.kaazing.gateway.management.gateway.ManagementGatewayStrategy;
+import org.kaazing.gateway.management.monitoring.configuration.MonitoringEntityFactoryBuilder;
+import org.kaazing.gateway.management.monitoring.configuration.impl.AgronaMonitoringEntityFactoryBuilder;
```

Comment:
impl classes should not be imported at this level.  Should use a factory for this.